packages/animations/test/open_container_test.dart

```diff
@@ -1951,7 +1966,9 @@ class __RemoveOpenContainerExampleState
                 ),
               ],
             ),
-            openBuilder: (BuildContext context, VoidCallback action) => Column(
+            openBuilder:
+                (BuildContext context, VoidCallback action, dynamic extras) =>
+                    Column(
```

Comment:
It looks like no test was added to actually test that the new functionality works as advertised.